src/rpc/RPCServer.ts

```diff
@@ -27,9 +26,9 @@ import {
   ServerHandler,
   UnaryHandler,
 } from './handlers';
-import * as rpcUtils from './utils';
+import * as rpcUtils from './utils/utils';
 import * as rpcErrors from './errors';
-import * as middlewareUtils from './middleware';
+import * as middlewareUtils from './utils/middleware';
 import { never } from '../utils/utils';
 import { sysexits } from '../errors';
 
```

Comment:
We need some docblock for `sensitive`. Well docblock in `createRPCServer` and explain what does `sensitive` mean.

Reply:
I've added docblocks to both `RPCClient` and `RPCServer` filling in some details.